Sources/SwiftParser/Expressions.swift

```diff
@@ -2544,6 +2574,52 @@ extension Parser {
   }
 }
 
+// MARK: Platform Condition
+extension Parser {
+  mutating func parseCanImportExpression(_ canImportKeyword: RawTokenSyntax) -> RawExprSyntax {
```

Comment:
The general paradigm in the parser is to pass the `TokenConsumptionHandle` for the `canImport` keyword instead of the keyword itself. Doesn’t really change much implementation-wise though.